hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestKeyManagerUnit.java

```diff
@@ -105,7 +104,6 @@ public class TestKeyManagerUnit {
   private KeyManagerImpl keyManager;
 
   private Instant startDate;
-  private File testDir;
```

Comment:
We can keep and annotate it, instead of adding parameter.

Reply:
It is not used anywhere else hence added as a parameter instead of keeping as instance variable. It is also removed from the method cleanup.